vllm/model_executor/layers/rotary_embedding.py

```diff
@@ -474,7 +474,7 @@ def get_rope(
     else:
         rope_scaling_args = None
     key = (head_size, rotary_dim, max_position, base, is_neox_style,
-           rope_scaling_args)
+           rope_scaling_args, torch.get_default_dtype())
```

Comment:
can we pass the dtype as an argument instead?

Reply:
done.